spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/MetricsAggregationCustomRepository.java

```diff
@@ -379,6 +376,28 @@ public long getNumberOfApiCallers(LocalDate from, LocalDate until, String apiPro
         return getSummarizedMetrics(serviceId, null, fromDate, untilDate);
     }
 
+    public List<StatisticsPerService> getStatisticsPerServiceByServicesOwnedByLoggedInUserAndDateRange(
+            List<ServiceApi> servicesOwnedByLoggedInUser,
+            LocalDate fromDate,
+            LocalDate untilDate) {
+
+        List<StatisticsPerService> listOfStatisticsPerServiceOwnedByLoggedInUser = new ArrayList<>();
+
+        servicesOwnedByLoggedInUser.forEach((service) -> {
+            List<MetricsAggregation> getSummarizedMetricsFilteredByMetricTypeResponse = getSummarizedMetrics(service.getId(), null, fromDate, untilDate).stream().filter(x -> x.getType() == MetricType.RESPONSE).collect(Collectors.toList());
```

Comment:
Please add line break.

Reply:
For better readability, please replace `x` by something like `metric`.

Reply:
Done